addon/-private/properties/alias.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+
+import {
+  getProperty,
+  objectHasProperty
+} from '../helpers';
+
+/**
+ * Returns the value of some other property on the PageObject.
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continue: alias('continueButton.click')
+ * });
+ *
+ * // calls `page.continueButton.click`
+ * page.continue();
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continueText: alias('continueButton.text')
+ * });
+ *
+ * // checks value of `page.continueButton.text`
+ * assert.equal(page.continueText, 'Go ahead!');
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   clickContinueButton: clickable('button.continue'),
+ *   continue: alias('clickContinueButton')
+ * });
+ *
+ * // calls `clickContinueButton`
+ * page.continue();
+ *
+ * @public
+ *
+ * @param {string} pathToProp - dot-separated path to PageObject property
```

Comment:
I think this description is very important to remark in the documentation.

What will be __supported__ by the add-on is the ability to point to other PageObject's property and not to any property.

```js
const myDescriptor = {
  isDescriptor: true,
  get(key) {
    return {
      hello: 'world'
    };
  }
};

const page = PageObject.create({
  foo: myDescriptor,

  aliased: alias('foo.hello')
});
```

Although this works, we should consider this as a _not supported_ scenario -- i.e. we don't promise this is going to continue working on a future version of the addon -- because `hello` is not a property of a page object, it's a property of a generic object returned by a page object's property.